### PR TITLE
Workflow status localization

### PR DESCRIPTION
**Pull request by monoludic**
Workflow status was not localized in some views.

### CODE REVIEW

File: src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs [hunk 1]
```diff
@@ -18,5 +18,22 @@ public static LocalizedHtmlString GetTitleOrDefault(this IActivity activity, Fun
             var title = activity.As<ActivityMetadata>().Title;
             return !string.IsNullOrEmpty(title) ? new LocalizedHtmlString(title, title) : defaultTitle();
         }
+
+        public static LocalizedHtmlString GetLocalizedStatus(this WorkflowStatus status, IViewLocalizer localizer)
```
Comment (jptissot) [inline src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs]: The POExtractor probably will not work in this case.

File: src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs [hunk 2]
```diff
@@ -19,20 +21,22 @@ public static LocalizedHtmlString GetTitleOrDefault(this IActivity activity, Fun
...
-                _ => localizer[status.ToString()],
+                WorkflowStatus.Aborted => H["Aborted"],
+                WorkflowStatus.Executing => H["Executing"],
+                WorkflowStatus.Faulted => H["Faulted"],
+                WorkflowStatus.Finished => H["Finished"],
+                WorkflowStatus.Halted => H["Halted"],
+                WorkflowStatus.Idle => H["Idle"],
+                WorkflowStatus.Resuming => H["Resuming"],
+                WorkflowStatus.Starting => H["Starting"],
+                _ => H[status.ToString()],
```
Comment (monoludic) [inline src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs]: @jptissot This works with PoExtractor, I tested.

Comment (jptissot) [reply src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs]: Yes, this looks good!

Comment (sebastienros) [reply src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs]: Just return `new LocalizedHtmlString(status)` in the default case.

**Comment by sebastienros** [comment]
Once my comment is applied I am fine merging it, thanks @monoludic

File: src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs (hunk 2 above)
Comment (monoludic) [reply src/OrchardCore/OrchardCore.Workflows.Abstractions/Helpers/ActivityExtensions.cs]: Fine.

**Pull request merged**